contracts/cw-distributor/src/contract.rs

```diff
@@ -0,0 +1,130 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{to_binary, CosmosMsg, Uint128, WasmMsg};
+
+use cosmwasm_std::{Binary, Deps, DepsMut, Env, MessageInfo, Response, StdResult};
+use cw2::set_contract_version;
+use stake_cw20::msg::ReceiveMsg;
+
+use crate::error::ContractError;
+use crate::msg::{ConfigResponse, ExecuteMsg, InstantiateMsg, LastPaymentBlockResponse, QueryMsg};
+use crate::state::{Config, CONFIG, LAST_PAYMENT_BLOCK};
+
+const CONTRACT_NAME: &str = "crates.io:cw-distributor";
```

Comment:
I would probable change name here and for file to `stake-cw20-reward-distributor` as I don't think the contract is very general purpose.

Reply:
this also make it clear it is part of the stake-cw20 family of contracts

Reply:
Done